wt-status.c

```diff
@@ -1003,16 +1003,17 @@ static void wt_longstatus_print_other(struct wt_status *s,
 
 size_t wt_status_locate_end(const char *s, size_t len)
 {
-	const char *p;
-	struct strbuf pattern = STRBUF_INIT;
+        const char *p;
+        struct strbuf pattern = STRBUF_INIT;
 
-	strbuf_addf(&pattern, "\n%c %s", comment_line_char, cut_line);
-	if (starts_with(s, pattern.buf + 1))
-		len = 0;
-	else if ((p = strstr(s, pattern.buf)))
-		len = p - s + 1;
-	strbuf_release(&pattern);
-	return len;
+        strbuf_addf(&pattern, "\n%c %s", comment_line_char, cut_line);
+
+        if (starts_with(s, pattern.buf + 1) && starts_with_newline(s + pattern.len - 1))
+                len = 0;
+        else if ((p = strstr(s, pattern.buf)) && starts_with_newline(p + pattern.len))
+                len = p - s + 1;
+        strbuf_release(&pattern);
+        return len;
```

Comment:
Changing the indentation is not only incorrect (and independent of the purpose of this PR), it also makes it a lot harder to spot your actual changes.